control_msgs/action/FollowJointTrajectory.action

```diff
@@ -1,5 +1,6 @@
 # The joint trajectory to follow
 trajectory_msgs/JointTrajectory trajectory
+trajectory_msgs/MultiDOFJointTrajectory multi_dof_trajectory
```

Comment:
Could use a comment above each of these types to explain what they do and what the difference is. Something like --
```

# For simple revolute or prismatic joints only
trajectory_msgs/JointTrajectory trajectory
# For all joint types, including floating and planar
trajectory_msgs/MultiDOFJointTrajectory multi_dof_trajectory
```

Reply:
Added comments